pkg/app/piped/cloudprovider/ecs/client.go

```diff
@@ -139,6 +142,7 @@ func (c *client) CreateTaskSet(ctx context.Context, service types.Service, taskD
 		Cluster:        service.ClusterArn,
 		Service:        service.ServiceArn,
 		TaskDefinition: taskDefinition.TaskDefinitionArn,
+		Scale:          &types.Scale{Unit: types.ScaleUnitPercent, Value: 100},
```

Comment:
should we replace this `100` value here with the real value passed by configuration 👀

Reply:
I missed it 👍

Reply:
Done with it. We use this field mainly in case using canary deployment. So, I'm going to implement traffic routing configuration in a new PR.

Reply:
Thanks 🙏